.github/workflows/generate.yml

```diff
@@ -33,23 +33,27 @@ jobs:
           go-version: '^1.17.3'
 
       - name: Generate CPE database
-        run: cd fleet && go mod download && go run -tags fts5 cmd/cpe/generate.go && cat ./etagenv >> $GITHUB_ENV
+        run: |
+          cd fleet
+          go mod download
+          go run -tags fts5 cmd/cpe/generate.go
+          cat ./etagenv >> $GITHUB_ENV
 
       # Note that a new commit must be made for each release, otherwise GitHub does not order
       # the releases properly.
       - name: Tag
-        if: hashFiles('fleet/*.sqlite.gz') != ''
```

Comment:
just double checking: why did you decide to remove this? seems like this step will be now always run, independently if new files were generated or not?

Reply:
This was redundant. If any previous step fails, then this step won't run. No need to override the if condition to check if the file was correctly generated.

Reply:
ah, makes sense